doc/reference.conf

```
@@ -546,12 +546,17 @@ operator "god" {
 	privset = "admin";
 };
 
-/* connect {}: controls servers we connect to (OLD C:, N:, H:, L:) */
+/* connect {}: controls servers we connect with (OLD C:, N:, H:, L:).
+ *
+ * This configuration is used whether connections are incoming or
+ * outgoing. */
```

Missed that this end-comment should be on a line by itself.  I'll make this change myself.

```suggestion
 * outgoing.
 */
```